mco_ovs_workers.yml.tmpl

```diff
@@ -3,7 +3,7 @@ kind: MachineConfig
 metadata:
   labels:
     machineconfiguration.openshift.io/role: worker
-  name: 00-configure-ovs-worker
+  name: 00-configure-interfaces-master
```

Comment:
Fix the name to worker:  00-configure-interfaces-master

Reply:
great catch!. Done